toolkit/tools/imagegen/installutils/installutils_test.go

```diff
@@ -0,0 +1,28 @@
+// Copyright Microsoft Corporation.
+// Licensed under the MIT License.
+
+package installutils
+
+import (
+	"runtime"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"microsoft.com/pkggen/internal/pkgjson"
+)
+
+func TestShouldSucceedRequiredPackages(t *testing.T) {
```

Comment:
Suggestion: I'd call it something closer to `TestShouldReturnCorrectRequiredPackagesForArch`. Now the name implies that we only care if getting the list of required packages succeeds, not necessarily that the returned list is correct.